test/Psr7ServerRequestTest.php

```diff
@@ -568,4 +568,13 @@ public function testPrivateConstruct()
         $this->expectExceptionMessage(sprintf('Call to private %s::__construct', Psr7ServerRequest::class));
         new Psr7ServerRequest();
     }
+
+    public function testFromLaminasCanHandleNullContent()
+    {
+        $laminasRequest = new LaminasRequest();
+        $laminasRequest->setContent(null);
+
+        $psr7Request = Psr7ServerRequest::fromLaminas($laminasRequest);
+        $this->assertEmpty($psr7Request->getBody()->getContents());
```

Comment:
The only thing I could note here is "strict type comparison" - maybe better will be use `assertSame(''` or `assertNull(` ?

Reply:
Strictness does not matter here. Assertion could have been omitted entirely and counter incremented manually as real assertion here is simply a lack of error.